### PR TITLE
Add support for versions

### PR DESCRIPTION
**Pull request by amatos**
created mackup/applications/versions.cfg, added versions entry to changelog.md

### CODE REVIEW

File: mackup/applications/versions.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = Versions
+
+[configuration_files]
+Library/Application Support/Versions/License
+Library/Preferences/com.blackpixel.versions.plist
```
Comment (lra) [inline mackup/applications/versions.cfg]: This lacks a newline character

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 - Added support for Bitchx (via @troywilson_)
 - Added support for EditorConfig (via @chadluo)
 - Add com.agilebits.onepassword4.plist in 1Password (via @amatos)
+- Added support for Versions (via @amatos)
```
Comment (lra) [inline CHANGELOG.md]: Can you add it to the README? See https://github.com/lra/mackup/blob/master/CONTRIBUTING.md

File: mackup/applications/versions.cfg (hunk 1 above)
Comment (amatos) [inline mackup/applications/versions.cfg]: Fixed, and hopefully committed correctly.

File: CHANGELOG.md (hunk 2 above)
Comment (amatos) [inline CHANGELOG.md]: Done.  My apologies.

**Comment by lra** [comment]
no worries ;)

**Pull request merged**